### ISSUE TITLE
Update PyPi with new setup.py file

### ISSUE THREAD

**Issue by mathDR** [marionmari/pyGPs#38]
Hey @marionmari @shansfolder can one of you update the pypi file for pygps?  I just added the future dependency in setup.py and now the code will compile and work from a git clone.  But I think it will still break if one tries to pip install pyGPs.

Thanks

**Comment by mathDR** [comment]
Was this done?  If so, we can close this

**Comment by shansfolder** [comment]
I just checked again. **pip install pyGPs** seems still broken for me. I will update the pypi file tomorrow and also start to take care of the other small open issue.

Sorry for being not very responsible on this project last year. Now I am back in the game! :)

**Comment by mathDR** [comment]
👍

**Comment by shansfolder** [comment]
Hi everyone, I made a new release 1.3.4 today which contains new commits from Sep 2016. It includes a few bugs fix, enhancement of the clustering demo for time series, and Python 3 future dependency fix in setup.py (described in this issue).

I tested the release using different python version and installation methods. It looks good. :)

[Here](https://github.com/marionmari/pyGPs/releases/tag/v1.3.4) is the new release and **pypi** is also up to date to the new release.
